### PR TITLE
Create configuring_prometheus.md

### PR DESCRIPTION
**Pull request by issmirnov**
Hey there,

Thanks for a really great project! This is a joy to use.

I put together some notes on how to set up exporting metrics to prometheus. Please take a look and let me know if you'd like any changes. 

Note: I'm also not sure how to add this under the "Guides" section in the sidebar - where is the index menu defined?

### CODE REVIEW

File: templates/docs/configuring_prometheus.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Configuring Prometheus
+
+Healthchecks.io supports exporting metrics and check statuses to [Prometheus](https://prometheus.io/), for use with [Grafana](https://grafana.com/).  
+
+You can genereate the metrics export endpoint by going to your project settings (last tab up top) and clicking "Create API Keys". You will then see the link to the prometheus endpoint, which looks like this:
```
Comment (cuu508) [inline templates/docs/configuring_prometheus.md:5]: typo: "genereate"

**Comment by cuu508** [comment]
Thanks for the PR!

There are similar instructions under "Project > Integrations > Add Prometheus" but I think a page in documentation would be useful as well.

The documentation sidebar is defined in https://github.com/healthchecks/healthchecks/blob/master/templates/front/base_docs.html

And you can generate html files from md by running `./manage.py render_docs`.

**Comment by issmirnov** [comment]
Done: fixed typo and ran the docs generator.

**Comment by cuu508** [comment]
Great, thanks!

I made a few additional changes:

* replaced the sample endpoint URL with a screenshot from the Project Settings screen
* replaced "https" and "healthchecks.io" with placeholders in the configuration example. The placeholders get replaced with actual values when serving the page: https://github.com/healthchecks/healthchecks/blob/master/hc/front/views.py#L281

**Comment by issmirnov** [comment]
Thank you! Love the improvements. Appreciate you catching the "promethus" typo as well.

**Pull request merged**